ubuntu/debian/libignition-gazebo.install

```diff
@@ -2,3 +2,5 @@ usr/lib/*/*.so.*
 usr/share/ignition/ignition-gazebo[0-99]/*.config
 usr/share/ignition/ignition-gazebo[0-99]/gui/*
 usr/share/ignition/ignition-gazebo[0-99]/worlds/*
+tools/desktop/ignition-gazebo.svg usr/share/icons/hicolor/scalable/apps/
```

Comment:
How about this file, it's not versioned, can it cause problems when installing multiple versions side-by-side?

Reply:
wow, good catch, yes it's going to conflict with other. Let me find a solution for it.

Reply:
https://github.com/ignitionrobotics/ign-gazebo/pull/1430 + change in [0edef93](https://github.com/ignition-release/ign-gazebo3-release/pull/16/commits/0edef93cae2d6a1f57cce5ad52876d1585b296b4) should make it.